src/formatting_stack/strategies/impl.clj

```diff
@@ -27,7 +31,9 @@
 
 (speced/defn ^::file-entries file-entries
   [& args]
-  (->> args (apply sh) :out string/split-lines (filter seq)))
+  ;;; xxx
+  (let [r (apply sh (conj (vec args) :dir *pwd*))]
+    (->> r :out string/split-lines (filter seq))))
```

Comment:
There should be a better way to add the `:dir` option here.

Reply:
I implemented a rudimentary approach where I use 1 impl for the `git`-command here; 15ba8e1

That suggestion also removes the `git-command`-def, and makes sure any git-command is executed in the right pwd. It removes /some/ flexibility from `impl/absolutize`, but we can extend/restore that if we ever need it in the future.

WDYT @vemv ?